.github/workflows/4-deploy-to-prod.yml

```
@@ -93,7 +93,7 @@ jobs:
         with:
           payload: |
             {
-              "text": "Production deploy **${{ steps.pages-deployment.outcome }}** for: ${{ github.ref_name }}"
+              "text": "Deploy finished for ${{ github.ref_name }}: *${{ steps.pages-deployment.outcome }}*"
```

nit: Same comment with `finished` as above
```suggestion
              "text": "Deploy for ${{ github.ref_name }} resulted in: *${{ steps.pages-deployment.outcome }}*"
```